docs/_includes/component.njk

```diff
@@ -416,14 +415,13 @@
   {% if component.dependencies.length %}
     <h2>Dependencies</h2>
 
-    <p>This component automatically imports the following dependencies.</p>
+<p>This component automatically imports the following dependencies.</p>
 
-    <ul>
-      {% for dependency in component.dependencies %}
-        <li>
-          <code>&lt;{{ dependency }}&gt;</code>
-        </li>
-      {% endfor %}
-    </ul>
-  {% endif %}
-{% endblock %}
\ No newline at end of file
+<ul>
+  {% for dependency in component.dependencies %}
+  <li>
+    <code>&lt;{{ dependency }}&gt;</code>
+  </li>
+  {% endfor %}
+</ul>
+{% endif %} {% endblock %}
```

Comment:
It would be good to figure out what's going on with these nesting changes; I don't think we actually want this not to be indented. Do you have a linter plugin running in VSCode that might be modifying when you save?

Reply:
I'm not sure what happened -- I had Nunjucks Template Formatter installed but maybe I didn't run it correctly? I just ran it again & it seems to have fixed this issue & looks like the indentation matches what's on Shoelace's latest github version of this file now? Just pushed this in a new commit